blog/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from .models import Post, User, Commentary
+from django.contrib.auth.models import Group
+
+admin.site.register(Post)
+admin.site.register(User)
+admin.site.register(Commentary)
+admin.site.unregister(Group)
+
+
+class PostAdmin(admin.ModelAdmin):
+    search_fields = ("created_time", "owner", "title", "content")
+    list_filter = ("created_time", "owner", "title", "content")
+
+
+class CommentaryAdmin(admin.ModelAdmin):
+    search_fields = ("created_time", "user", "post", "content")
+    list_filter = ("created_time", "user", "post", "content")
```

Comment:
Using 'content' in 'list_filter' might not be optimal if 'content' is a large text field. Consider removing it from 'list_filter'.